src/Entity/MemberDetailsRevision.php

```diff
@@ -79,6 +86,7 @@ public function __construct(Member $member, bool $own) {
         $this->postCode = $member->getPostCode();
         $this->country = $member->getCountry();
         $this->dateOfBirth = $member->getDateOfBirth();
+        $this->currentMembershipStatus = $member->getCurrentMembershipStatus();
     }
 
     public function hasChanged(Member $member) {
```

Comment:
Don't you need to add a check in this function to see if the membershipstatus has changed? Otherwise it won't be recorded, right?